### ISSUE TITLE
Cannot run Tailscale after update to DSM 7

### ISSUE THREAD

**Issue by Markmaster** [tailscale/tailscale#3234]
I was on DSM 6. Updated to the DSM 7. Uninstall Tailscale. After the update, I got this error while trying to run the start command: `sudo tailscale up`:
`sudo tailscale up
failed to connect to local tailscaled (which appears to be running). Got error: Failed to connect to local Tailscale daemon for /localapi/v0/status; not running? Error: dial unix /var/packages/Tailscale/etc/tailscaled.sock: connect: connection refused`
Before upgrading from DSM 6 I delete TailScale, after updating to DSM 7 I first try to install TailScale from Synology Store, but the version is outdated, and If I try to Open the app while I'm on the same local network with Synology NAS, I got this page:
[https://cln.sh/f2iLAS](url)

Synology DS416, DSM 7.0-41890
Have no luck with CLI: [https://cln.sh/cT93fU](url)

**Comment by DentonGentry** [comment]
You might be running into https://tailscale.com/kb/1152/synology-outbound/?q=DSM7
That knowledge base article explains how to enable outbound connections.

-----

Alternately, if you are comfortable loading software from outside the package center,  https://github.com/tailscale/tailscale-synology has an updated version which has fixed this issue. We get an opportunity to update the Package Center once a month, the 1.16.x release on https://github.com/tailscale/tailscale-synology will be our December update.

**Comment by DentonGentry** [comment]
Closing as https://tailscale.com/kb/1152/synology-outbound/?q=DSM7

**Comment by camaz** [comment]
I'm having and identical issue with a newly upgraded DS1517+ to DSM 7.0. Uninstalled Tailscale, upgraded DSM to 7 successfully, and reinstalled Tailscale from the Package Center. No luck, so I side-loaded the Tailscale 1.18.0 SPK from github, but same result: 
`failed to connect to local tailscaled (which appears to be running). Got error: Failed to connect to local Tailscale daemon for /localapi/v0/status; not running? Error: dial unix /var/packages/Tailscale/etc/tailscaled.sock: connect: connection refused`
I also enabled outbound connections as directed at the link posted above, to no effect :(

**Comment by camaz** [comment]
> does /var/packages/Tailscale/etc/tailscaled.sock exist?

Yes

> is a process named tailscaled running? What arguments does it have?

`/volume1/@appstore/Tailscale/bin/tailscaled --state=/volume1/@appdata/Tailscale/tailscaled.state --socket=/volume1/@appdata/Tailscale/tailscaled.sock --port=41641`

**Comment by DentonGentry** [comment]
Does this command succeed?
`/volume1/@appstore/Tailscale/bin/tailscale --socket=/volume1/@appdata/Tailscale/tailscaled.sock up`

Was your Synology device installed a long time ago with a much older DSM release, and then repeatedly upgraded until now it is running DSM7?

I think that original release put package center binaries in a different place, and Synology keeps the original path.

**Comment by camaz** [comment]
> Does this command succeed?
> /volume1/@appstore/Tailscale/bin/tailscale --socket=/volume1/@appdata/Tailscale/tailscaled.sock up

Yes, I was able to login via the url provided on the command line.

> Was your Synology device installed a long time ago with a much older DSM release, and then repeatedly upgraded until now it is running DSM7?

Most likely just before the DSM 6 release landed.. Perhaps it's a good time for a re-image of DSM.

**Comment by camaz** [comment]
I uninstalled all packages, and reinstalled DSM 7 with the Mode 2 reset described [here](https://kb.synology.com/en-global/DSM/tutorial/How_to_reset_my_Synology_NAS_7). After initial configuration, I installed tailscale from the official Synology repo, but still encountered the original error. I was, however, able to start tailscaled from the command line, and it now seems to be working as normal. Hope this helps anyone else who encounters this error..

@DentonGentry thanks for the response!
